tools/Dashboard/DevHome.Dashboard/Controls/WidgetControl.xaml

```diff
@@ -16,23 +16,25 @@
             <ResourceDictionary.ThemeDictionaries>
                 <ResourceDictionary x:Key="Light">
                     <Color x:Key="WidgetBorderColor">#0F000000</Color>
+                    <Color x:Key="WidgetBackgroundColor">#B4FFFFFF</Color>
                 </ResourceDictionary>
                 <ResourceDictionary x:Key="Dark">
                     <Color x:Key="WidgetBorderColor">#1A000000</Color>
+                    <Color x:Key="WidgetBackgroundColor">#0CFFFFFF</Color>
                 </ResourceDictionary>
             </ResourceDictionary.ThemeDictionaries>
         </ResourceDictionary>
     </UserControl.Resources>
 
     <Grid Width="300" Height="{x:Bind helpers:WidgetHelpers.GetPixelHeightFromWidgetSize(WidgetSource.WidgetSize), Mode=OneWay}" 
-          CornerRadius="7" BorderBrush="{ThemeResource WidgetBorderColor}" BorderThickness="1">
+          CornerRadius="7" BorderBrush="{ThemeResource WidgetBorderColor}" Background="{ThemeResource WidgetBackgroundColor}" BorderThickness="1">

```

Comment:
I think it's worth investigating. I have to admit I find certain xaml styling standards harder to read (ie. if we always keep parameters on a line under the element name like the image in that link, that aligns those parameters with any contained elements, which I find confusing. Also if parameters comfortably fit on one line, I don't like wasting space with newlines). But that's just me and it can be a discussion.

Reply:
IMO, readability trumps all, as readability is maintainability, and easier to spot errors.